src/main/kotlin/eu/kevin/api/models/general/bank/BankResponse.kt

```diff
@@ -12,5 +12,8 @@ data class BankResponse(
     val isSandbox: Boolean,
     val imageUri: String? = null,
```

Comment:
Looking at the code, I am a little bit confused why `imageUri` and `officialName` properties are nullable. Can it even happen?

Reply:
I believe those should not be nullable.